impl/client/src/main/doc-js/pentaho/visual/_namespace.jsdoc

```diff
@@ -15,7 +15,48 @@
  */
 
 /**
- * The `visual` namespace contains the classes and interfaces that constitute the Visualization API.
+ * The `visual` namespace contains the types that constitute the **Pentaho Visualization API**.
+ *
+ * ## Overview
+ *
+ * The Pentaho Visualization API provides a unified way to visualize data across the Pentaho suite
+ * (e.g. [Analyzer]{@link http://www.pentaho.com/product/business-visualization-analytics},
+ *       [PDI]{@link http://www.pentaho.com/product/data-integration},
+ *       [CDF]{@link http://community.pentaho.com/ctools/cdf/}).
+ * Essentially, it is a set of abstractions that ensures total isolation between
+ * applications,
+ * visualizations and
+ * configurations (the glue).
+ *
+ * Visualizations are constituted by one
+ * [Model]{@link pentaho.visual.base.Model} and (at least) one [View]{@link pentaho.visual.base.View}.
+ * Models identify visualizations and define their data requirements.
+ * Views implement the actual rendering using chosen technologies
+ * (e.g. [HTML]{@link https://www.w3.org/TR/html/},
+ *       [SVG]{@link https://www.w3.org/TR/SVG/},
+ *       [D3]{@link https://d3js.org/}),
+ * and handle user interaction,
+ * dispatching [actions]{@link pentaho.visual.action} and, for example, showing tooltips.
+ * The standard data actions are
+ * [Select]{@link pentaho.visual.action.Select} and
+ * [Execute]{@link pentaho.visual.action.Execute}.
+ *
+ * Visualizations are implemented on top of the Pentaho Core, Type and Data JavaScript APIs:
+ * * Using the [Type API]{@link pentaho.type} endows visualizations with out-of-the-box class inheritance,
+ *   metadata support, type configuration, validation, serialization, among other features.
+ * * Using the [Data API]{@link pentaho.data} ensures seamless integration with data sources in the Pentaho platform,
+ *   as well as with other client-side component frameworks.
+ * * Using [Core APIs]{@link pentaho} provides visualizations with features such as localization,
+ *   theming and registration.
+ *
+ * The use of [Pentaho Platform JavaScript APIs]{@link pentaho} for anything not directly related with the
+ * visualization concept, ensures developers can reuse knowledge that they gain in other components of the
+ * Pentaho suite to build visualizations and vice-versa.
+ *
+ * The Pentaho Visualization API comes with a set of stock visualizations,
+ * covering the most common chart-types.
+ * Most stock visualizations are based on the [CCC]{@link http://community.pentaho.com/ctools/ccc/} charting library,
+ * which means that the stock visualizations are _super-configurable_ to suite your organization's preferred style.
```

Comment:
This text seems to mimic the one from the walkthrough... So maybe update it when the walkthrough is finalised.